Formula/s2geometry.rb

```diff
@@ -0,0 +1,91 @@
+class S2geometry < Formula
+  desc "Computational geometry and spatial indexing on the sphere"
+  homepage "https://github.com/google/s2geometry.git"
+  url "https://github.com/google/s2geometry/archive/v0.9.0.tar.gz"
+  sha256 "54c09b653f68929e8929bffa60ea568e26f3b4a51e1b1734f5c3c037f1d89062"
+
+  depends_on "cmake" => :build
+  depends_on "glog" => :build
+  depends_on "ninja" => :build
```

Comment:
`ninja` is required but it's not clear where it's used

Reply:
removed